src/test_utils/mod.rs

```diff
@@ -103,7 +103,8 @@ impl<'a> OneShotAuroraRunner<'a> {
         );
 
         near_vm_runner::run(
-            &self.base.code,
+            self.base.code.hash.as_ref().to_vec(),
+            &self.base.code.code.as_slice(),
```

Comment:
`base.code.code` seems a bit... confusing :P

Reply:
Yes, it was nicer before for sure. This became necessary because I needed to use an older version of `near-vm-logic` so that it matches the version used by `near-sdk-sim`.